weather-forecast/gravity-wave-drag/cnns/configs/cnn.yaml

```diff
@@ -29,6 +29,6 @@ fit:
     
     trainer:
         max_epochs: 10
-        accelerator: "cpu"
+        accelerator: "gpu"
         devices: 
             - 0
```

Comment:
Some remarks concerning the devices as before

Reply:
Same answer :)